testing/checkpolicy/APKBUILD

```diff
@@ -0,0 +1,30 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=checkpolicy
+pkgver=2.6
+pkgrel=0
+pkgdesc="SELinux policy checker"
+url="http://www.selinuxproject.org"
+arch="all"
+license="GPLv2"
+depends="flex-dev"
```

Comment:
This looks suspicious. Packages usually don’t depend on -dev (i.e. .h .a, .la, … files). Are you sure that checkpolicy needs flex-dev, not just flex?